lib/helper.coffee

```diff
@@ -0,0 +1,91 @@
+ChildProcess = require 'child_process'
+Path = require 'path'
+FS = require 'fs'
+{XRegExp} = require 'xregexp'
+
+class Helpers
+  @findFile: (directory, fileName) ->
```

Comment:
What if I told you that using any of the functions as a callback will break if you use `this`?

Since none of these hold state, there's no reason to require the user to call them with the right context.